pkg/interface/chat/src/js/store.js

```diff
@@ -18,6 +18,7 @@ class Store {
       spinner: false,
       sidebarShown: true,
       pendingMessages: new Map([]),
+      myReadReceiptsPending: new Map([]),
```

Comment:
This is truly terrible naming I think, but couldn't come up with anything better! This is a map of station -> boolean, indicating whether we are waiting on a response to our own read receipt request.

Reply:
I've been sitting here for a few minutes now thinking about it. `pendingReadReceipts`?